Containers/imaginary/Dockerfile

```diff
@@ -0,0 +1,2 @@
+# From https://github.com/h2non/imaginary/blob/master/Dockerfile
+FROM h2non/imaginary:1.2.4

```

Comment:
this release and image is two years old and needs to be updated for it to work, IIRC.